crates/viewer/re_viewer_context/src/test_context.rs

```diff
@@ -72,11 +81,122 @@ impl Default for TestContext {
             reflection,
             command_sender,
             command_receiver,
+
+            // Created lazily since each egui_kittest harness needs a new one.
+            egui_render_state: Mutex::new(None),
         }
     }
 }
 
+fn create_egui_renderstate() -> Option<egui_wgpu::RenderState> {
```

Comment:
it would be helpful with a docstring explaining why this could return `None`

Reply:
on its way out, but will do anyways just in case I can't finish my thread of work